js/index.js

```diff
@@ -0,0 +1,18 @@
+
+let today = new Date();
+let thisYear = today.getFullYear();
+let footer = document.querySelector('.footer');
+let copyright = document.createElement('p')
+copyright.innerHTML = "Emmely " + thisYear;
+footer.appendChild(copyright);
+
+
+var skills = ["HTML","CSS","JavaScript"];
+var skillsSection = document.getElementById('mySkills');
+var skillsList = skillsSection.querySelector('ul');
```

Comment:
```suggestion
var skillsList = document.querySelector('#mySkills ul');
```
What you have works, but another option is just to put it all in one selector.